src/data_utils/ProcessTrafficData.py

```diff
@@ -51,7 +54,7 @@ def LoadTrafficData(dataset_path, segment, time_from, time_to):
         [x_min - 50, x_max + 50, y_min - 50, y_max + 50, time_from, time_to])
 
     traffic_data = {}
-    for data in cur.fetchall():
+    for data in tqdm(cur.fetchall()):
```

Comment:
Add a progress bar for the functions in processtrafficdata. This comes to handy when processing many segments, and not really being sure if it is still running.